frontend/src/pages/Registration.js

```diff
@@ -16,7 +16,9 @@ export default function Registration() {
 
   const handleSubmit = async (value) => {
     try {
-      const response = await signUp(value)
+      let type
+      value.taxNumber === '' ? (type = 'INDIVIDUAL') : (type = 'COMPANY')
```

Comment:
Huh, ilyet ne csináljunk, több okból:
1. 1 percig néztem mire megértettem mi történik és miért.
2. A type inicializálatlan, nincs default értéke, később inicializáljuk, nem a deklarációnál. Ez rosszabb esetben elmaradhat, ha változik a kód.
3. A ternary kifejezés értékét eldobjuk, valamint így is nehéz látni mi lenne az.
4. Az értékadás megtörténik, de ez egy mellékhatás egy kifejezés kiértékelése során.

Vagy
`let type = value.taxNumber === '' ? 'INDIVIDUAL' : 'COMPANY'`
vagy
```
let type = 'COMPANY'

if(value.taxNumber === '') {
    type = 'INDIVIDUAL'
}
```